### PR TITLE
Fix mongodb daemon shutdown

### PR DESCRIPTION
**Pull request by jhadvig**
Shutting down mongodb daemon more clearly

### CODE REVIEW

File: 2.4/run-mongod.sh [hunk 1]
```diff
@@ -71,7 +71,7 @@ function create_mongodb_users() {
 
 	# Create standard user with read/write permissions, in specified database ('production' by default).
 	mongo $mongo_db --eval "db.addUser({user: '${mongo_user}', pwd: '${mongo_pass}', roles: [ 'readWrite' ]});"
-	mongo admin --eval "db.shutdownServer();"
+	mongod -f /opt/openshift/etc/mongodb.conf --shutdown
```
Comment (mfojtik) [inline 2.4/run-mongod.sh]: make this env var. don't use paths directly

File: 2.4/run-mongod.sh [hunk 2]
```diff
@@ -7,6 +7,8 @@ set -e
 MAX_ATTEMPTS=60
 SLEEP_TIME=1
 
+MONGODB_CONFIG_PATH="/opt/openshift/etc/mongodb.conf"
```
Comment (mfojtik) [inline 2.4/run-mongod.sh:10]: `WORKDIR /opt/openshift`

this means that you can just do:

MONGODB_CONFIG_PATH="./etc/mongodb.conf"

Comment (jhadvig) [inline 2.4/run-mongod.sh:10]: @mfojtik imo that would only confuse some users. And if somebody would want to change the path of the config file, he would need to do it even in Dockerfile

Comment (mfojtik) [inline 2.4/run-mongod.sh:10]: k

**Pull request merged**